licenses/management/commands/publish.py

```diff
@@ -0,0 +1,148 @@
+    return subprocess.run(PULL_DOWN_TRANSLATION_BRANCHES, shell=True, check=True,)
+
+
+def list_open_branches():
+    """List of open branches in cc-licenses-data repo
+    """
+    list_branches_cmd = GO_TO_TRANSLATIONS_REPO + "git branch --list"
+    pull_translations_branches()
+    branches = (
+        subprocess.check_output(list_branches_cmd, shell=True).decode().split("\n")
+    )
+    print("\n\nWhich branch are we publishing to?\n")
+    [print(b) for b in branches]
+
+
+def publish_branch(branch: str):
+    """Workflow for publishing a single branch"""
+    git_on_branch_and_pull(branch)
+    run_django_distill()
+    build_to_push = git_branch_status(branch)
+    if build_to_push:
+        git_commit_and_push(branch)
+    else:
+        print(f"\n{branch} build dir is up to date.\n")
+
+
+def publish_all():
+    """Workflow for checking branches other than develop and updating their build dir
+
+    Develop is not checked because it serves as the source of truth. It should be
+    manually merged into.
+    """
+    list_branches_cmd = GO_TO_TRANSLATIONS_REPO + "git branch --list"
+    branches = (
+        subprocess.check_output(list_branches_cmd, shell=True).decode().split("\n")
+    )
+    exclude_develop_list = [b for b in branches if "develop" not in b]
+    del exclude_develop_list[-1]
+    branch_list = [b.lstrip() for b in exclude_develop_list]
+    print(
+        f"\n\nChecking and updating build dirs for {len(branch_list)} translation branches"
+    )
+    [publish_branch(b) for b in branch_list]
+
+
+class Command(BaseCommand):
+    """Command to push the static files in the build directory to a specified branch
+    in cc-licenses-data repository
+
+    Arguments:
+        branch_name - Branch name in cc-license-data to pull translations from
+                      and publish artifacts too.
+        list_branches - A list of active branches in cc-licenses-data will be
+                        displayed
+
+    If no arguments are supplied all cc-licenses-data branches are checked and
+    then updated.
+    """
+
+    def add_arguments(self, parser: ArgumentParser):
+        parser.add_argument(
+            "-b",
+            "--branch_name",
+            help=(
+                "Branch name in cc-license-data to pull translations from "
+                "and push artifacts too. If not specified a list of active "
+                "branches in cc-licenses-data will be displayed"
+            ),
+        )
+        parser.add_argument(
+            "-l",
+            "--list_branches",
+            action="store_true",
+            help="A list of active branches in cc-licenses-data will be displayed",
+        )
+
+    def handle(self, *args, **options):
+        if options.get("list_branches"):
+            return list_open_branches()
+        if options.get("branch_name"):
+            return publish_branch(options["branch_name"])
+        return publish_all()
```

Comment:
Generally Django isn't expecting the handle() method to return anything.

Reply:
updated to use if/elif/else